mkdocs.yml

```diff
@@ -70,6 +70,7 @@ markdown_extensions:
 copyright: Copyright &copy; 2021 Amazon Web Services
 
 plugins:
+  - git-revision-date
```

Comment:
I don't see how it can work... we removed it because of an error (https://github.com/awslabs/aws-lambda-powertools-java/issues/855). There's nothing in the PR that makes me think it will be different now...

Reply:
I'm still doing some experiments. Including testing the actions/setup-python@v4. However, the flow works on branch execution.

Reply:
I've added the build doc workflow, looks like it passes... I guess we can move on or are you still testing things?

Reply:
Yes, it's ready. Thanks for your help.

Reply:
why did you remove what I've added ? I think that make sense to test the doc build when mkdocs changes...

Reply:
Also be careful with forced pushes, can be useful sometimes but looks like you do it each time.